spec/functional/win32/security_spec.rb

```diff
@@ -149,4 +150,32 @@
       end
     end
   end
+
+  describe "self.get_account_right" do
```

Comment:
Usually the name for this describe block would be ".get_account_right" for a class method. Eg: http://www.betterspecs.org/#describe